### PR TITLE
Update roadmap

### PR DESCRIPTION
**Pull request by mapedraza**
The purpose of this PR is to update the roadmap

- [x] ~~Add new tasks?~~
- [x] Move completed tasks
- [x] Add release reference to completed tasks

### CODE REVIEW

**Comment by fgalan** [comment]
> This section has been last updated in January 2020. Please take into account its  content could be obsolete.

It should be March 2022

**Comment by mapedraza** [comment]
> 138921f

Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/2145/commits/138921ff91f3a4a8d4c7c73c5e54f461556883d0

File: doc/roadmap.md [hunk 1]
```diff
@@ -32,34 +32,29 @@ Disclaimer:
...
+- Ability to disable metadata (all sinks) [#1706](https://github.com/telefonicaid/fiware-cygnus/issues/1706) ([1.18.0](https://github.com/telefonicaid/fiware-cygnus/releases/tag/1.18.0))
+- dm-by-entitytype in additional sinks (PostGIS and PostgreSQL sinks) [#1684](https://github.com/telefonicaid/fiware-cygnus/issues/1684) ([2.0.0](https://github.com/telefonicaid/fiware-cygnus/releases/tag/2.0.0))
+- Improve the error feedback provided to users (all sinks) [#1791](https://github.com/telefonicaid/fiware-cygnus/issues/1791) ([2.0.0](https://github.com/telefonicaid/fiware-cygnus/releases/tag/2.0.0))
+- Operational requirements (logs, correlation id, etc.) (all sinks) [#1770](https://github.com/telefonicaid/fiware-cygnus/issues/1770) ([2.0.0](https://github.com/telefonicaid/fiware-cygnus/releases/tag/2.0.0))
+- New mode dm-by-entity-shifted (CKAN sink) [#1792](https://github.com/telefonicaid/fiware-cygnus/issues/1792) ([2.2.0](https://github.com/telefonicaid/fiware-cygnus/releases/tag/2.2.0))
```
Comment (fgalan) [inline doc/roadmap.md:60]: I think is better inverse chronological order, as in Orion (https://github.com/telefonicaid/fiware-orion/blob/master/doc/roadmap.md#features-already-completed)

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**